### PR TITLE
Fixed : "IE 11 ignores dialogWidth-Height" #74 (2) … _NoPostback method does not work because the return value of dialog is not returned.

### LINKED ISSUE #74: IE 11 ignores dialogWidth-Height
# Summary
IE 9 ignores postback possible dialog's Width-Height properties by using Frameset.

## Solution
- The solution is to add following at page that contains the frameset.

 &lt;meta http-equiv = "x-ua-compatible" content = "IE = 7"&gt;

- Target
  - OpenTouryoProject-OpenTouryo-DialogFrame.htm  
https://github.com/OpenTouryoProject/OpenTouryo/blob/develop/root/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Aspx/Framework/DialogFrame.htm

## Problem
This issue's modification not yet started by following reasons.
- There is possibility that IE will be modified for this problem.
- Future support for showModalDialog method.
  - Document modes are deprecated (Windows)  
https://msdn.microsoft.com/en-us//library/ie/dn384051.aspx
  - Chromium Blog Disabling showModalDialog  
http://blog.chromium.org/2014/07/disabling-showmodaldialog.html

## Reference
- size - Internet Explorer 9 ignores dialogWidth-Height for showModalDialog with a frameset - Stack Overflow  
http://stackoverflow.com/questions/9067667/internet-explorer-9-ignores-dialogwidth-height-for-showmodaldialog-with-a-frames
- IEバージョンアップ情報 - マイクロソフト系技術情報 Wiki  
  - IE9へ > IE9 移行時の問題点事例 > IE9でダイアログのサイズ指定が出来ない。  
http://techinfoofmicrosofttech.osscons.jp/index.php?IE%E3%83%90%E3%83%BC%E3%82%B8%E3%83%A7%E3%83%B3%E3%82%A2%E3%83%83%E3%83%97%E6%83%85%E5%A0%B1#zb6941d0
  - モーダルダイアログのサポート状況  
http://techinfoofmicrosofttech.osscons.jp/index.php?IE%E3%83%90%E3%83%BC%E3%82%B8%E3%83%A7%E3%83%B3%E3%82%A2%E3%83%83%E3%83%97%E6%83%85%E5%A0%B1#m2a42571

### PR DESCRIPTION
**Pull request by daisukenishino**
fixed #74
- [Fix of window.returnValue issue that is occurred by IFRAME of IE11 correspondance. #105](https://github.com/SymphonyTeleca/OpenTouryo/pull/105)
- [Merged the issue of IE 11 ignores dialogWidth-Height from C# version #107](https://github.com/SymphonyTeleca/OpenTouryo/pull/107)

### CODE REVIEW

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Please merge this pull requests ASAP.

File: root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js [hunk 1]
```diff
@@ -734,6 +735,8 @@ function Fx_ShowModalScreen(url, style) {
         // 第2引数 = DialogFrame → DialogLoader.htmから起動するモーダル画面のURL
         // 第3引数 = 画面のスタイル(「項目1:値1;項目2:値2;…;項目n:値n」の形式) 
         var ret = window.showModalDialog(dialogFrameUrl.value, args, style);
+        ret = Fx_GetCookie("fx_window_returnValue");
```
Comment (yusukemaegawa) [inline root/programs/C#/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js]: @daisukenishino 

At lines 737, you assign the return value of showModalDialog method to the variable ret.
Immediately after this, you assign the return value of Fx_GetCookie method to the variable ret again.

It is meaningless that you assign the return value of showModalDialog method to variable ret at lines 737.

File: root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js [hunk 2]
```diff
@@ -167,6 +169,15 @@ function Fx_OnSubmit() {
...
             // IE8.0では完全に有効
         }
+        else if (navigator.appVersion.indexOf("MSIE 9.0") != -1) {
+            // IE9.0で問題の報告を受けていません。 
+        }
+        else if (navigator.appVersion.indexOf("MSIE 10.0") != -1) {
+            // IE10.0で問題の報告を受けていません。 
+        }
+        else if (navigator.appVersion.indexOf("Trident/7") != -1) {
+            // IE11.0で問題が合った場合、報告をお願いします。
```
Comment (yusukemaegawa) [inline root/programs/VB/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js]: @daisukenishino 

「問題が合った」 -> 「問題があった」

**Comment by daisukenishino** [comment]
@yusukemaegawa 

>https://github.com/OpenTouryoProject/OpenTouryo/pull/160#discussion_r56259746

registered.
see : https://github.com/OpenTouryoProject/OpenTouryo/issues/149#issuecomment-197089822

>https://github.com/OpenTouryoProject/OpenTouryo/pull/160#discussion-diff-56259874

fixed.
see : https://github.com/OpenTouryoProject/OpenTouryo/pull/160/files

Please merge this pull requests ASAP.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

Modification does not correspond to the title of this pull request.
Please review the title of this pull request.

**Comment by daisukenishino** [comment]
@yusukemaegawa 

I fixed the title.
Please merge.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

The code that substitutes empty value for window.returnValue still remains.
Is the modification of this code necessary?

[common.js L38](https://github.com/SymphonyTeleca/OpenTouryo/blob/nishino/root/programs/C%23/Samples/WebApp_sample/ProjectX_sample/Framework/Js/common.js#L38)
```js
function Fx_Document_OnLoad2() {
    window.returnValue = "";
```

**Comment by daisukenishino** [comment]
@yusukemaegawa 

I will not modify this code,
because it is not clear whether this correcpondence is for the "business modal dialog".

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>I will not modify this code,
>because it is not clear whether this correcpondence is for the "business modal dialog".

Well, why does this code exist?

**Comment by daisukenishino** [comment]
@yusukemaegawa 

>Well, why does this code exist?

It is unknown, I remembered again that comments are too important.
Because it is the unknown, I think that it should not be removed at this time.

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>It is unknown, I remembered again that comments are too important.
>Because it is the unknown, I think that it should not be removed at this time.

If so, why don't you research the meaning of this code?
(You may create new issue.)

**Comment by daisukenishino** [comment]
@yusukemaegawa 

see : https://github.com/OpenTouryoProject/OpenTouryo/issues/149#issuecomment-197089822

**Comment by yusukemaegawa** [comment]
@daisukenishino 

>see : https://github.com/OpenTouryoProject/OpenTouryo/issues/149#issuecomment-197089822

I confirmed the above.

**Pull request merged**